lab-kyler/greet.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+module.exports = {};
+
+if (process.argv[2]) console.log('Hello, ' + process.argv[2] + '!');
+
+module.exports.greet = function(name) {
+  if (!name) throw new Error('No name provided!');
+  return 'Hello, ' + name + '!';
+};
```

Comment:
Glad to see the correct use of module.exports to export a single function! As a minor nitpick, I would start trying to use string interpolation where appropriate as it's something that we will be using a lot in this course.

ie: return 'Hello, ' + name +'!'        --->       return `Hello ${name}`